.github/workflows/build-and-deploy-on-pypi.yml

```diff
@@ -0,0 +1,55 @@
+name: PyPi Build and Deploy 🐍📦
+
+on:
+  release:
+    types: [published]
+  # use this for testing
+  push:
+    branches:
+      - main
+      - add_pypi_pkg_build_GA
+
+jobs:
+  build-n-publish:
+    name: Build and publish MyProxyClient on PyPi
+    runs-on: ubuntu-latest
+    # may also need this for publishing?
+    environment:
+      name: pypi
+      url: https://pypi.org/project/MyProxyClient/
```

Comment:
Why is this needed?

Reply:
the instructions at that site in the comment say we should add that - not sure of it myself, but I thought I'd follow instructions ad literam